package.json

```diff
@@ -46,6 +46,8 @@
     "babel-istanbul": "^0.6.0",
     "babel-loader": "^6.2.3",
     "babel-plugin-transform-object-rest-spread": "^6.6.5",
+    "babel-plugin-transform-runtime": "^6.9.0",
+    "babel-polyfill": "^6.9.1",
```

Comment:
`babel-polyfill` has no documentation ... https://www.npmjs.com/package/babel-polyfill
![babel-polyfill-no-readme](https://cloud.githubusercontent.com/assets/194400/16380538/8a7f94dc-3c6f-11e6-8f5c-40f5f7f09899.png)

I'm sure the blog post or StackOverflow answer that instructs to use `babel-polyfill` to solve the IE Compatibility issue is convincing, but I don't _understand_ what it _does_ ... 
tried reading the code on Github: https://github.com/babel/babel/tree/master/packages/babel-polyfill 
Appears to import https://github.com/zloirock/core-js
Then what?
What is the effect on the size of `bundle.js` from importing a global JS Shim?

> This _appears_ to be the _only_ way of "_fixing_" the problem, so we should _probably_ just use it, but I want to _understand_ the impact ... 😕